package.json

```diff
@@ -51,7 +51,9 @@
     }
   },
   "dependencies": {
-    "electron-squirrel-startup": "^1.0.0"
+    "electron-squirrel-startup": "^1.0.0",
+    "react-router-bootstrap": "^0.25.0",
+    "react-router-dom": "^5.2.0"
```

Comment:
React router bootstrap and react-router DOM only need to be in `react-app/package.json` they can be uninstalled from the root package.json